working-with-fhir-igs/hl7v2-to-dk-medcom/extensions.bal

```diff
@@ -0,0 +1,184 @@
+import ballerina/io;
```

Comment:
Let's remove if not used

Reply:
fixed in [4a378ec](https://github.com/ballerina-guides/healthcare-samples/pull/38/commits/4a378ec4f64c338c69ab412bcd053c7d95901093)